lib/src/components/NxFileUpload/SelectedFile.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { forwardRef } from 'react';
+import { faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import prettyBytes from 'pretty-bytes';
+
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
+import NxTooltip from '../NxTooltip/NxTooltip';
+import { SelectedFileProps } from './types';
+
+export { SelectedFileProps };
+
+const formatSize = (size: number) => prettyBytes(size, { minimumFractionDigits: 1, maximumFractionDigits: 1 });
+
+const SelectedFile = forwardRef<HTMLSpanElement, SelectedFileProps>(function
```

Comment:
Not a fan of this formatting.  I'd put a line break after the opening `(`, then put the function signature on a single line, and indent the entire function as needed (2 levels) and then separate the closing `}` and `);` onto two separate lines with the `);` not indented

Reply:
> Not a fan of this formatting. I'd put a line break after the opening `(`, then put the function signature on a single line, and indent the entire function as needed (2 levels) and then separate the closing `}` and `);` onto two separate lines with the `);` not indented

Reformatted here: [65c90a0](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/65c90a0674bb975fa2559ab758bd32d1c38eef9b)